tests/system/action/motion_comment/test_update.py

```diff
@@ -158,9 +158,21 @@ def test_update_permission_cause_submitter(self) -> None:
 
     def test_update_permission_non_meeting_committee_admin(self) -> None:
         self.set_committee_management_level([60])
-        self.test_models["committee/60"] = {"user_ids": [1], "manager_ids": [1]}
         self.base_update_permission_non_meeting_admin()
 
+    def test_create_permission_non_meeting_parent_committee_admin(self) -> None:
+        self.create_meeting()
+        self.create_committee(committee_id=61, parent_id=60)
+        self.set_committee_management_level([61])
+        self.test_models.update(
+            {
+                "committee/60": {"meeting_ids": []},
+                "committee/61": {"meeting_ids": [1]},
+                "meeting/1": {"committee_id": 61},
+            }
+        )
+        self.base_update_permission_non_meeting_admin(meeting_exists=True)
```

Comment:
See also the create test.

```
        self.create_committee(59)
        self.create_committee(60, parent_id=59)
        self.set_committee_management_level([60])
        self.base_update_permission_non_meeting_admin()
```

Creates basically the same structure, although I am guessing that you want to test what happens when the action is called as a parent committees manager. in that case it'd be

```suggestion
        self.create_committee(59)
        self.create_committee(60, parent_id=59)
        self.set_committee_management_level([59])
        self.base_update_permission_non_meeting_admin()
```